environment.sh

```diff
@@ -115,8 +96,8 @@ if [ $update_env == 'no' ]; then
     repo=${repo_info[0]}
     repo_branch=${repo_info[2]}
     last_commit_time=$(curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/ihmeuw/$repo/commits?sha=$repo_branch | jq '.["0"].commit.committer.date')
-    if [[ $requirements_modification_time < $last_commit_time ]]; then
-      update_env="yes"
+    if [[ $requirements_modification_time < $last_commit_time ]] && [[ $creation_time > $last_commit_time ]]; then
```

Comment:
Why are you doing the first check? Don't we only need to check that `[[ $creation_time > $last_commit_time ]]`?